docs/content/tutorial/disaster-recovery.md

```diff
@@ -231,7 +231,22 @@ you can start accessing your database from that specific point in time!
 
 ## Restore Individual Databases
 
-You can restore individual databases using a spec similar to the following:
+You might need to restore specific databases from a cluster backup, for performance reasons
+or to move selected databases to a machine that does not have enough space to restore the
+entire cluster backup.
+
+{{% notice warning %}}
+pgBackRest supports this case, but it is important to make sure this is what you want.
+Restoring in this manner will restore the requested database from backup and make it
+accessible, but all of the other databases in the backup will NOT be accessible after restore.
+
+For example, if your backup includes databases `test1`, `test2` and `test3`, and you request that
```

Comment:
⛏️ (nitpick, not essential). I'm not sure what the rest of our docs look like, but I slightly prefer
```suggestion
For example, if your backup includes databases `test1`, `test2`, and `test3`, and you request that
```

Reply:
Added this change